### PR TITLE
NEUBURG-30: Add config for smpt, database url

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "flow.pathToFlow": "${workspaceRoot}/node_modules/.bin/flow",
+  "flow.useNPMPackagedFlow": true,
+  "javascript.validate.enable": false,
+  "yarn.packageJson": "package.json",
+  "jira.baseUrl": "https://integreat.atlassian.net/",
+  "jira.projectNames": "NEUBURG"
+}
```
Comment (NKlug) [inline .vscode/settings.json:8]: Do we want this Visual Studio Code file in the repo?

Comment (maxammann) [reply .vscode/settings.json:8]: Yes, this configures some useful/needed plugins of vscode

**Review by NKlug — APPROVED** [review]

**Review by steffenkleinle — CHANGES_REQUESTED** [review]
https://github.com/Integreat/neuburg-backend/pull/6/files?utf8=%E2%9C%93&diff=unified#diff-453452d78c4c37b73382bc89a66d51cbR39
smtp config missing in const mailService = new MailService()

**Comment by maxammann** [comment]
@klinzo done

**Review by steffenkleinle — APPROVED** [review]

File: src/services/OfferService.js [hunk 2]
```diff
@@ -33,48 +45,60 @@ export default class OfferService {
...
-      .where('expirationDate').gt(Date.now())
-      .where('deleted').equals(false)
-      .populate({path: 'formData', select: '-_id -__v'})
+      .where('city')
+      .equals(city)
+      .where('expirationDate')
+      .gt(Date.now())
+      .where('deleted')
+      .equals(false)
+      .populate({ path: 'formData', select: '-_id -__v' })
```
Comment (michael-markl) [inline src/services/OfferService.js:61]: Is it forced, to have new lines when starting '.' (probably only when they overflow)?
I think the structure was better before.

**Review by michael-markl — APPROVED** [review]

**Pull request merged**